birdhouse/pavics-compose.sh

```diff
@@ -25,11 +24,17 @@ VARS='
   $POSTGRES_PAVICS_PASSWORD
   $POSTGRES_MAGPIE_USERNAME
   $POSTGRES_MAGPIE_PASSWORD
+  $SERVER_SYNOPSIS
```

Comment:
The default values of all these in `default.env` should then be empty if they are to be in `VARS` list.

Recall settings in `VARS` list are the settings we want to force users to set a value because no default value can be appropriate.  So if the default value in `default.env` is not empty, then it defeats the check that user has set a personalized value.